tests/lib/at_custom_cmd/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  at_custom_cmd.functionality_test:
+    platform_allow: nrf9160dk_nrf9160_ns
```

Comment:
Please add integration platform option to make it possible to build this test in Jenkins CI:
```
    integration_platforms:
      - nrf9160dk_nrf9160_ns
```